llvm/test/CodeGen/DirectX/splitdouble.ll

```diff
@@ -1,5 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
 ; RUN: opt -passes='function(scalarizer)' -S -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s
+; RUN: opt -passes='function(scalarizer),module(dxil-op-lower)' -S -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s --check-prefix=DXIL-LOWER
```

Comment:
`test_vector_double_split_void` also needs a DXIL-LOWER prefix.

Also don't you want to test the scalar case?